contribs/gnodev/pkg/browser/model_nav.go

```diff
@@ -0,0 +1,73 @@
+package browser
+
+import (
+	"fmt"
+	"strings"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/glamour"
+)
+
+func (m *model) moveToRealm(realm string) tea.Cmd {
+	path := cleanupRealmPath(m.urlPrefix, realm)
+
+	// Set uri input
+	m.urlInput.SetValue(path)
+	m.urlInput.CursorEnd()
+
+	// return command update
+	return tea.Sequence(RefreshRealm(), m.urlInput.Focus())
+}
+
+func (m *model) updateHistory() {
+	v := m.urlInput.Value()
+	if m.history.Len() == 0 {
+		m.current = m.history.PushBack(v)
+		return
+	}
+
+	m.current = m.history.InsertAfter(v, m.current)
+	for next := m.current.Next(); next != nil; {
+		m.history.Remove(next)
+		next = m.current.Next()
+	}
+}
+
+func (m *model) moveHistoryForward() (string, bool) {
+	if next := m.current.Next(); next != nil {
+		m.current = next
+		return m.current.Value.(string), true
+	}
+	return "", false
+}
+
+func (m *model) moveHistoryBackward() (string, bool) {
+	if prev := m.current.Prev(); prev != nil {
+		m.current = prev
+		return m.current.Value.(string), true
+	}
+	return "", false
+}
+
+func (m model) fetchRenderView(path string) (view []byte, err error) {
```

Comment:
I think this is some kind of troll, every last method in each file is a named return

Reply:
Since I'm in the minority for the named return wild west with @gfanton and @deelawn, I will resolve and go back to my linter cave 🏇